src/utilstrencodings.cpp

```diff
@@ -227,8 +227,8 @@ std::vector<unsigned char> DecodeBase64(const char* p, bool* pfInvalid)
 
 std::string DecodeBase64(const std::string& str)
 {
-    std::vector<unsigned char> vchRet = DecodeBase64(str.c_str());
-    return (vchRet.size() == 0) ? std::string() : std::string((const char*)&vchRet[0], vchRet.size());
+    vector<unsigned char> vchRet = DecodeBase64(str.c_str());
```

Comment:
In commit "Fix 0-subscript in utilstrencodings.cpp"

Should put back std:: prefixes.